lib/v8-to-istanbul.js

```diff
@@ -27,11 +28,13 @@ module.exports = class V8ToIstanbul {
     this.sourceTranspiled = undefined
   }
   async load () {
-    const rawSource = readFileSync(this.path, 'utf8')
-    // if we find a source-map (either inline, or a .map file) we load
-    // both the transpiled and original source, both of which are used during
-    // the backflips we perform to remap absolute to relative positions.
-    const rawSourceMap = convertSourceMap.fromSource(rawSource) || convertSourceMap.fromMapFileSource(rawSource, dirname(this.path))
+    const rawSource = this.sources.source || readFileSync(this.path, 'utf8')
```

Comment:
When `this.sources.source` is a zero-length string, it still tries to read an actual file. I guess you meant

```js
this.sources.source === undefined ? readFileSync(this.path, 'utf8') : this.sources.source
```

Reply:
good call!

FWIW, I'm still not happy with the branching logic I introduced in this PR, I'd prefer an explicit `.loadFromFs` and `.loadFromMemory` or something API. Haven't had time to dig further into this yet though

Reply:
👍 Separate APIs really sound good to me.